website/docs/tutorials/07-testing/03-fuzzing/strategies.md

```diff
@@ -75,6 +75,20 @@ Fuzzer picks integers from provided range and then converts them to felts.
 If `min_value` is not `None` then all values will be greater than or equal to `min_value`,
 and if `max_value` is not `None` then all values will be less than or equal to `max_value`.
 
+### `strategy.short_strings`
+```python
+def text(
+    alphabet: Union[Sequence[str], SearchStrategy[str]] = characters(
+        blacklist_categories=("Cs",)
+    ),
+    min_size: int = 0,
+    max_size: Optional[int] = None,
+) -> Strategy:
+```
+
+Generates strings with ascii lowercase characters of length that passes the condition `0 <= length <= 31`.
+Max size `31` comes from [the docs](https://www.cairo-lang.org/docs/how_cairo_works/consts.html#short-string-literals).
```

Comment:
Maybe callout ::info ?

Reply:
Added in [ab28a1a](https://github.com/software-mansion/protostar/pull/886/commits/ab28a1ad8001eadf597047acc3e9553478ca918b)